packages/forklift-console-plugin/src/modules/Providers/views/list/ProvidersListPage.tsx

```diff
@@ -130,7 +135,30 @@ export const fieldsMetadataFactory: ResourceFieldFactory = (t) => [
   },
   {
     resourceFieldId: 'storageCount',
-    jsonPath: '$.inventory.storageCount',
+    jsonPath: (obj: ProviderData) => {
```

Comment:
@yaacov 
the fields cannot be all set at the same time - types are mutually exclusive.
We could leverage that in the similar way as the original FlatProvider merge function:
https://github.com/kubev2v/forklift-console-plugin/blob/cbd3ff3ab3524f0ce475748e0f22617d65dcfd5b/packages/forklift-console-plugin/src/modules/Providers/data.ts#L160